src/modules/web-inbox/WebInbox.js

```diff
@@ -415,6 +416,7 @@ export class Inbox extends HTMLElement {
       this.inboxCard.scrollTop = 0
       !this.isPreview && this.deleteExpiredAndGetUnexpiredMsgs()
       this.inbox.style.display = 'block'
+      this.inbox.style.zIndex = '2147483647'
```

Comment:
@KambleSonam this number is hardcoded, why? explain in a comment

Reply:
max value for zIndex is 2147483647. For our inbox to render above all elements it should have the highest zIndex value.

Reply:
ok, please add it as a comment in the code so that a new developer can understand as well. 😇